ci/bullseye_generate_report.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# This is the script used for generating the Bullseye Reports stage in CI
+set -ex
```

Comment:
Haven't we standardised on variables needing to be bound in our `ci/*` scripts?

```suggestion
set -eux
```